docs/api-reference.rst

```diff
@@ -4,6 +4,37 @@ API Reference
 Many of these classes are missing methods from the SQLALchemy API. We encourage
 you to :doc:`file bugs </contributing>` in those cases.
 
+.. currentmodule:: alchimia
+
+.. function:: wrap_engine(reactor, engine, create_worker=...)
+
+   This returns a :py:class:`alchimia.engine.TwistedEngine`.
+
+   The main entry-point to alchimia.  To be used like so::
+
+        from sqlalchemy import create_engine
+        from alchimia import wrap_engine
+        from twisted.internet import reactor
+
+        underlying_engine = create_engine("sqlite://")
+        twisted_engine = wrap_engine(reactor, engine)
+
+   - ``reactor`` - the Twisted reactor to use with the created
+     ``TwistedEngine``.
+
+   - ``engine`` - the underlying :py:class:`sqlalchemy.engine.Engine`
+
+   - ``create_worker`` - The object that will coordinate concurrent blocking
+       work behind the scenes.  The default implementation, if nothing is
+       passed, is one which will use a threadpool where each Connection is tied
+       to an individual thread.
+
+       More precisely, this is a callable that is expected to return an object
```

Comment:
Maybe link to [IWorker](https://twistedmatrix.com/documents/current/api/twisted._threads.IWorker.html)?  Or not because it's still in `_threads._ithreads`?

Reply:
Yeah, it's a simple enough interface I'd rather just duplicate it here than go through that hassle.